settings.gradle

```diff
@@ -131,15 +129,8 @@ include 'data-prepper-plugins:opensearch-source'
 include 'data-prepper-plugins:user-agent-processor'
 include 'data-prepper-plugins:in-memory-source-coordination-store'
 include 'data-prepper-plugins:aws-plugin-api'
-include 'data-prepper-plugins:aws-plugin'
 include 'data-prepper-plugins:obfuscate-processor'
 include 'data-prepper-plugins:parquet-codecs'
-include 'data-prepper-plugins:aws-sqs-common'
```

Comment:
I think we can keep this one. The `s3` source may already be using it.

Reply:
I can keep it. But I checked and it's only being used in `sqs` source

Reply:
OK. I thought it was supposed to consolidate the SQS logic. Apparently it does no such thing.